docker/Dockerfile

```diff
@@ -34,19 +36,27 @@ ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/openmpi/lib
 
 ENV NCCL_LAUNCH_MODE=GROUP
 
+#install nccl
+RUN wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/cuda-keyring_1.0-1_all.deb && \
+    dpkg -i cuda-keyring_1.0-1_all.deb && \
+    apt update && \
+    apt install libnccl2 libnccl-dev
+
 # Should be in the lmdeploy root directory when building docker image
 COPY . /opt/lmdeploy
 
 WORKDIR /opt/lmdeploy
 
 RUN cd /opt/lmdeploy &&\
+    apt-get install -y git &&\
```

Comment:
installing git should put at the top of RUN command

Reply:
i will commit a new pr